src/auth/utils/getIdentityInformation.ts

```
@@ -43,18 +44,35 @@ const result: IIdentityInfo = {
 	twitter: '',
 	verifiedByPolkassembly: false,
 	web: ''
 };
 
+const getParentProxyInfo = async ({ address, api, apiReady, network }: { address: string; api: ApiPromise; apiReady: boolean; network: string }) => {
+	if (!api || !apiReady) return { address: '', title: null };
+	const encodedAddress = getEncodedAddress(address, network) || address;
+
+	const proxyInfo = await api?.query?.identity?.superOf(encodedAddress);
+	const formatedProxyInfo: any = proxyInfo?.toHuman();
+	if (formatedProxyInfo && formatedProxyInfo?.[0] && getEncodedAddress(formatedProxyInfo?.[0] || '', network)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)